client/account-recovery/lost-password/lost-password-form/test/index.jsx

```diff
@@ -24,22 +23,22 @@ describe( 'LostPassword', () => {
 		useFakeDom();
 
 		it( 'submit button shuold be disabled if user login is blank', function() {
-			const wrapper = mount( <LostPasswordFormComponent className="test__test" /> );
+			const wrapper = mount(
+				<LostPasswordFormComponent className="test__test"
+					userLogin={ '' }
+				/> );
```

Comment:
Would this break our long line rule if it were just

```
const wrapper = mount( <LostPasswordFormComponent className="test__test" userLogin="" /> );
```

or

```
const wrapper = mount(
    <LostPasswordFormComponent className="test__test" userLogin="" />
);
```

Also I'm not sure if you need the `{ '' }` if your just using a string literal.